featurebyte/api/credential.py

```diff
@@ -22,15 +22,6 @@
 class Credential(DeletableApiObject, SavableApiObject):
     """
     Credential class is the data model used to represent your credentials that are persisted.
-
-    Examples
-    --------
-    >>> credential = Credential(  # doctest: +SKIP
-    ...   name=feature_store.name,
-    ...   feature_store_id=feature_store.id,
-    ...   database_credential=database_credential,
-    ...   storage_credential=storage_credential,
-    ... )
```

Comment:
Remove the example here as the interface has been changed to `Credential.create`.